crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -978,6 +958,75 @@ impl DataLoadTracing {
     }
 }
 
+// This function is moved from RefreshTask for the testing.
```

Comment:
```suggestion
```

This could just be a comment in the PR, no need to keep it as a permanent comment in the code.